packages/experimental/Button/src/ToggleButton/ToggleButton.styling.windows.ts

```diff
@@ -0,0 +1,70 @@
+import { toggleButtonName, ToggleButtonTokens, ToggleButtonSlotProps, ToggleButtonProps } from './ToggleButton.types';
+import { Theme, UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { borderStyles, fontStyles } from '@fluentui-react-native/tokens';
+import { defaultButtonTokens } from '../ButtonTokens';
+import { buttonStates } from '../Button.styling';
+
+export const stylingSettings: UseStylingOptions<ToggleButtonProps, ToggleButtonSlotProps, ToggleButtonTokens> = {
+  tokens: [
+    defaultButtonTokens,
+    (t: Theme): ToggleButtonTokens => ({
+      checked: {
+        color: t.colors.neutralForeground1,
+        backgroundColor: t.colors.neutralBackground1Selected,
+        hovered: {
+          color: t.colors.neutralForeground1,
+          backgroundColor: t.colors.neutralBackground1Hover,
+        },
+        subtle: {
+          color: t.colors.neutralForeground1,
+          backgroundColor: t.colors.neutralBackground1Selected,
+          hovered: {
+            color: t.colors.neutralForeground1,
+            backgroundColor: t.colors.neutralBackground1Hover,
+            borderColor: t.colors.neutralStroke1,
+          },
+        },
+      },
+    }),
+    toggleButtonName,
+  ],
+  states: ['checked', ...buttonStates],
+  slotProps: {
+    root: buildProps(
+      (tokens: ToggleButtonTokens, theme: Theme) => ({
+        style: {
+          display: 'flex',
+          alignItems: 'center',
+          flexDirection: 'row',
+          alignSelf: 'flex-start',
+          justifyContent: 'center',
+          minHeight: 32,
+          minWidth: 80,
+          width: tokens.width,
```

Comment:
Does 'width' need to be added to the array on line 50 like 'backgroundColor' is?

Reply:
Good catch. I'll go update the styling on the component.